app/json/slate_configs.json

```diff
@@ -89,6 +89,24 @@
       }
     ]
   },
+  {
+    "id": "f99178fb-6bd0-4fa1-8109-cda181b697f6",
+    "displayName": "New Tab Recommendations",
```

Comment:
I didn't mention en-us in the displayName or the description because if we're displaying those somewhere, readers do not need to know about feeds. I put it in the internalDescription instead. However, that means the slate titles and descriptions for all 5 feeds will be similar/the same and the internalDescription will be the differentiator. Is that in line with our desired conventions?

Reply:
Joel may be the main consumer of the displayName